projects/management/commands/import_projects.py

```diff
@@ -52,20 +62,37 @@ def import_project(self, row, row_num):
         )
         p.save()
 
-    def import_csv(self, filename):
+    def parse_file(self, filename):
         with open(filename, encoding='utf-8') as csvfile:
             reader = csv.DictReader(csvfile)
 
             # Skip first row
             next(reader)
 
             for i, row in enumerate(reader):
-                self.import_project(row, i + 1)
+                self.add_project(row, i + 1)
+
+    def import_s3(self, filename):
+        s3 = boto3.resource(
+            's3',
+            aws_access_key_id=settings.AWS_ACCESS_KEY_ID,
+            aws_secret_access_key=settings.AWS_SECRET_ACCESS_KEY,
+        )
+
+        with NamedTemporaryFile() as f:
+            s3.meta.client.download_file(settings.AWS_BUCKET, filename, f.name)
+            return self.parse_file(f.name)
+
+    def load(self, filename, s3):
```

Comment:
Since `s3` is a boolean, should we call it `use_s3` just to make that super obvious (and e.g. not make the reader wonder if it's some kind of object that represents an S3 connection)?